### PR TITLE
Update cp-quickstart for KRaft to run ksqlDB

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

Update cp-quickstart for KRaft which has ksqlDB elements now

Sister PR to https://github.com/confluentinc/cp-all-in-one/pull/104

_What behavior does this PR change, and why?_

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

For anyone testing this, please don't forget to update the branch that the script pulls from

```
-wget -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-kraft/docker-compose.yml
-wget -O update_run.sh https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-kraft/update_run.sh
+wget -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/add-kraft-ksql/cp-all-in-one-kraft/docker-compose.yml
+wget -O update_run.sh https://raw.githubusercontent.com/confluentinc/cp-all-in-one/add-kraft-ksql/cp-all-in-one-kraft/update_run.sh
```

### CODE REVIEW

File: cp-quickstart/start-docker-kraft.sh [hunk 1]
```diff
@@ -19,8 +19,10 @@ sleep 2 # give connect an exta moment to fully mature
 echo "connect has started!"
 
 # Configure datagen connectors
-source ./connectors/submit_datagen_pageviews_config.sh
-source ./connectors/submit_datagen_users_config.sh
+wget -O connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
```
Comment (davetroiano) [inline cp-quickstart/start-docker-kraft.sh]: Why not use the scripts here anymore? The kraft quickstart seems to work ok using the scripts, i.e., with this change:
```
-wget -O connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
-curl -X POST -H "Content-Type: application/json" --data @connector_pageviews_cos.config http://localhost:8083/connectors
-wget -O connector_users_cos.config https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_users_cos.config
-curl -X POST -H "Content-Type: application/json" --data @connector_users_cos.config http://localhost:8083/connectors
+source ./connectors/submit_datagen_pageviews_config.sh
+source ./connectors/submit_datagen_users_config.sh
```

Comment (ybyzek) [reply cp-quickstart/start-docker-kraft.sh]: Yeah, but that one has interceptors, and C3 wasn't part of the docker-compose.  But see other comment, do we want C3 in or out?  We can add it in if you think it's helpful

Comment (davetroiano) [reply cp-quickstart/start-docker-kraft.sh]: chatted IRL about this, result being that you're going to add C3

**Comment by ybyzek** [comment]
@davetroiano please make one more pass when you have a chance, I believe this is ready for merge.  The cp-all-in-one PR has already been merged

**Review by davetroiano — APPROVED** [review]
LGTM, smoke tested CP quickstart and C3. thanks @ybyzek!

**Comment by ybyzek** [comment]
Thanks @davetroiano 🎉

**Pull request merged**